x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/crawl_details_flyout/crawl_details_summary.tsx

```diff
@@ -0,0 +1,261 @@
+                    )}
+                  />
+                </EuiText>
+              }
+            />
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiStat
+              data-test-subj="pagesVisited"
+              titleSize="s"
+              title={
+                stats && stats.status && stats.status.pagesVisited
+                  ? stats.status.pagesVisited
+                  : '--'
+              }
+              description={
+                <EuiText size="s">
+                  {i18n.translate(
+                    'xpack.enterpriseSearch.appSearch.crawler.crawlDetailsSummary.pagesVisitedTooltipTitle',
+                    {
+                      defaultMessage: 'Pages',
+                    }
+                  )}{' '}
+                  <EuiIconTip
+                    type="iInCircle"
+                    color="primary"
+                    size="m"
+                    title={i18n.translate(
+                      'xpack.enterpriseSearch.appSearch.crawler.crawlDetailsSummary.pagesTooltipTitle',
+                      {
+                        defaultMessage: 'Pages visited',
+                      }
+                    )}
+                    content={i18n.translate(
+                      'xpack.enterpriseSearch.appSearch.crawler.crawlDetailsSummary.pagesTooltip',
+                      {
+                        defaultMessage: 'URLs visited and extracted during the crawl.',
+                      }
+                    )}
+                  />
+                </EuiText>
+              }
+            />
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiStat
+              data-test-subj="avgResponseTime"
+              titleSize="s"
+              title={
+                stats && stats.status && stats.status.avgResponseTimeMSec
+                  ? `${Math.round(stats.status.avgResponseTimeMSec)}ms`
+                  : '--'
+              }
+              description={i18n.translate(
+                'xpack.enterpriseSearch.appSearch.crawler.crawlDetailsSummary.avgResponseTimeLabel',
+                {
+                  defaultMessage: 'Avg. response',
+                }
+              )}
+            />
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiStat
+              data-test-subj="clientErrors"
+              titleSize="s"
+              title={statusCounts.clientErrorCount}
+              description={`4xx ${i18n.translate(
+                'xpack.enterpriseSearch.appSearch.crawler.crawlDetailsSummary.clientErrorsLabel',
+                {
+                  defaultMessage: 'Errors',
+                }
+              )}`}
+            />
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiStat
+              data-test-subj="serverErrors"
+              titleSize="s"
+              title={statusCounts.serverErrorCount}
+              description={`5xx ${i18n.translate(
```

Comment:
Ditto

Reply:
Did this for you to keep things moving: https://github.com/elastic/kibana/pull/120187/commits/92e52f59a2b1ef9558217634dcc4b8b17ac5b37e